optuna_dashboard/artifact/_backend.py

```diff
@@ -44,6 +44,16 @@
 )  # 128MB
 
 
+def get_artifact_url(
```

Comment:
`get_artifact_path` or `get_artifact_endpoint_path` might be better since it does not contain the scheme and the domain name.

```suggestion
def get_artifact_path(
```